src/AddNodeToggleSwitch.scss

```diff
@@ -0,0 +1,37 @@
+@import 'variables';
+
+.AddNodeToggleSwitch__switch {
```

Comment:
We can remove the "switch" Element part from the `AddNodeToggleSwitch__switch` class name as it refers to the whole component. `AddNodeToggleSwitch__switch-inner-circle` is good as it refers to a part of the component.

So:

- `AddNodeToggleSwitch__switch` -> `AddNodeToggleSwitch`
- `AddNodeToggleSwitch__switch--checked` -> `AddNodeToggleSwitch--checked`
- and so on